apps/src/componentLibrary/dropdown/checkboxDropdown/CHANGELOG.md

```diff
@@ -2,7 +2,14 @@
 
 All notable changes to this project will be documented in this file.
 
-## [0.6.0] (https://github.com/code-dot-org/code-dot-org/pull/62023)
+## [0.7.0](https://github.com/code-dot-org/code-dot-org/pull/62910)
```

Comment:
We might have some overlap here w/ [this version update](https://github.com/code-dot-org/code-dot-org/pull/62917/files#diff-dfbbeac300e664e3325a24b3abc37a195fab783bc779fe14786e0b7c98194313) from the color variable swap.